setup.sh

```diff
@@ -8,8 +8,14 @@ set -e
 
 # Run each JavaScript file in synchronous order
 
+echo "You have uncommited changes, stashing the changes to proceed with the pull"
+git stash
+
 git pull
 
+echo "Popping the stash to reapply the changes"
+git stash pop
+
```

Comment:
this may cause merge conflict and stop the script from running. that's my main concern.

Reply:
hmm yeah, maybe only stash changes and echo that changes been stashed

Reply:
removes stash pop